include/wlr/wayland.h

```diff
@@ -10,8 +10,8 @@ struct wlr_output_mode_state;
 struct wlr_output_mode {
 	struct wlr_output_mode_state *state;
 	uint32_t flags; // enum wl_output_mode
-	int32_t width, height;
-	int32_t refresh; // mHz
+	uint32_t width, height;
+	uint32_t refresh; // mHz
```

Comment:
Yeah, it makes absolutely no sense for them to be negative, so I changed them. They can be changed back to signed though; it wouldn't hurt anything.